third_party/move/tools/move-formatter/tests/formatter/use/input5.fmt.move

```diff
@@ -0,0 +1,12 @@
+/// test_point: Multiple blank lines between use statements
+
+module BlockLine {
+    // Multiple blank lines between statements
+    use aptos_std::type_info::{
+        /* use_item before */ Self,
+        TypeInfo};
```

Comment:
`}` should be in a new line?

Reply:
Yes, it makes sense

Reply:
The situation here is caused by comments and blank lines. For more common examples, it is not an issue. You can refer to this new use case: third_party/move/tools/move-formatter/tests/formatter/use/input6.fmt.move.

Reply:
Yes, it's difficult, there's no general solution here. But fortunately, in most cases, users would not make such complex use statement blocks. Various other formatting tools may also exhibit unreasonable behavior in such scenarios, which can be observed in their respective issue.